src/box/schema_def.h

```diff
@@ -131,6 +130,9 @@ enum {
 };
 /** \endcond public */
 
+/** Max possible space id. */
+extern uint64_t BOX_SPACE_MAX;
```

Comment:
2^31-1 and 2^31-2 are in the `int32_t` (and `uint32_t`) range. Why the tweak type is `uint64_t`?

Is the idea to allow to relax the limit in a future if we'll rewrite all the related APIs? I guess, if we do, the tweak type is the simplest part.

OTOH, I see no problems in such implementation and I think that 64 bit tweaks may be useful in a future. So, if you want, let's leave it as is.

Reply:
True, we could use `uint32_t` for `BOX_SPACE_MAX` but `TWEAK_UINT` requires `uint64_t`. I could introduce a new helper `TWEAK_UINT32` that would work for `uint32_t` but I don't want to because wherever we need `uint32_t` `uint64_t` will do as well.